hexrd/imageseries/load/imagefiles.py

```diff
@@ -219,6 +224,9 @@ def __init__(self, filename, **kwargs):
             self._fabioclass = img.classname
             self._imgframes = img.nframes
             self.dat = img.data
+            if os.path.splitext(filename)[-1] == '.gel':
+                raw_img = np.invert(img.data)
```

Comment:
Why is the data inverted here and not above in the __get_item__() method?

Reply:
I checked with @bnmajor, and it should be inverted in both. I'm modifying this logic to be in a function instead.